Mod Manager/src/routes/modList/+page.svelte

```diff
@@ -550,24 +644,83 @@
 <Modal passiveModal open={frameworkDeployModalOpen} modalHeading="Applying your mods" preventCloseOnClickOutside>
 	Your mods are being deployed. This may take a while - grab a coffee or something.
 	<br />
-	<pre
-		class="mt-2 h-[10vh] overflow-y-auto whitespace-pre-wrap bg-neutral-800 p-2"
-		style="font-family: 'Fira Code', 'IBM Plex Mono', 'Menlo', 'DejaVu Sans Mono', 'Bitstream Vera Sans Mono', Courier, monospace; color-scheme: dark"
-		id="deployOutputElement">{@html deployOutputHTML}</pre>
-	{#if deployOutput.split(/\r?\n/).some((a) => a.match(/.*WARN.*?\t/)) || deployOutput.split(/\r?\n/).some((a) => a.match(/.*ERROR.*?\t/))}
-		<br />
-		<div class="flex flex-row gap-2 flex-wrap max-h-[15vh] overflow-y-auto">
-			{#each deployDiagnostics as line}
-				<InlineNotification hideCloseButton lowContrast kind={line.includes("WARN") ? "warning" : "error"}>
-					<div slot="title" class="-mt-1 text-lg">
-						{line.includes("WARN") ? "Warning" : "Error"}
-					</div>
-					<div slot="subtitle">{line.replace(/.*WARN.*?\t/, "").replace(/.*ERROR.*?\t/, "")}</div>
-				</InlineNotification>
-			{/each}
+    <div 
+		id="deployOutputElement"
+		class="mt-2 h-[40vh] overflow-y-auto bg-neutral-800 p-2"
+        style="font-family: 'Fira Code', 'IBM Plex Mono', 'Menlo', 'DejaVu Sans Mono', 'Bitstream Vera Sans Mono', Courier, monospace; color-scheme: dark"
+		on:scroll={(event) => handleScroll(event, 'deployOutputElement')}>
+
+        {#each outputLines as line}
+            <pre class="whitespace-pre-wrap break-words text-gray-600">{@html convertAnsi.toHtml(line)}</pre>
+        {/each}
+
+        {#if userHasScrolledDeployOutput}
+            <button 
+                class="absolute top-[288px] z-10 right-[37px] bg-gray-400 hover:bg-red-500 text-white py-1 px-3 rounded"
```

Comment:
Do these absolute styles work the same across the common resolutions (720p, 1080p, 1440p, 2160p)? You can test using responsive mode in DevTools if you haven't already